### DISCUSSION TITLE
Is it possible to use a template engine i.e. Pug?

### DISCUSSION THREAD

**Discussion by kenanyusuf** [codesandbox/sandpack#303] [Q&A] [answered]
Hey, I'm experimenting with Sandpack and I was wondering whether it's possible to use a Node template engine like Pug? Parcel seems to support Pug but after experimenting with a few different approaches I haven't been able to get it to work.

Here's my attempt: https://codesandbox.io/s/wizardly-lichterman-gy1ll?file=/src/App.js:0-892

I based the Sandpack setup above on this CodeSandbox https://codesandbox.io/s/qkp0ojk8pj?file=/package.json:271-276

Appreciate any thoughts!

Cheers

**Comment by DeMoorJasper** [comment, marked as answer]
Currently we don't support any template languages, we might add this in the future

**Reply by kenanyusuf** [reply]
Thanks for the quick reply and for clarifying.

Just curious, are there technical limitations as to why template languages are not supported or is it just not a priority?

**Reply by DeMoorJasper** [reply]
It's mainly not a priority right now.

It's also not a small change, to support it properly we need a better way to handle html, we can probably use iframes for it, which will likely happen at some point as we want to separate execution from bundling more anyway
